### PR TITLE
doc: add known issue for ISO HCI command.

### PR DESCRIPTION
**Pull request by henrla**
Document known issue for LE Read ISO TX Sync.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -3222,6 +3222,13 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
+.. rst-class:: v2.6.0
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:3225]:
```suggestion
.. rst-class:: v2-6-0
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -3222,6 +3222,13 @@ SoftDevice Controller
...
+
+DRGN-21293: Le Read ISO TX Sync is implemented according to the raised errata ES-23138.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:3227]:
```suggestion
DRGN-21293: The LE Read ISO TX Sync command is implemented according to the raised errata ES-23138. 
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -3222,6 +3222,13 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
+.. rst-class:: v2.6.0
+
+DRGN-21293: Le Read ISO TX Sync is implemented according to the raised errata ES-23138. 
+  In ES-23138, the return parameter TX_Time_Stamp is modified to be the SDU synchronization reference of the SDU previously scheduled for transmission, and not the CIG or BIG reference anchor point.
+  Note that there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference when the CIG or BIG is configured with an ISO_Interval that equals the SDU_Interval.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:3229]:
```suggestion
  Note that when the CIG or BIG is configured with an ``ISO_Interval`` that equals the ``SDU_Interval``, there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference .
```
Alternatively, these could be just ISO interval and SDU interval, and not be parameters at all.

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 4]
```diff
@@ -3222,6 +3222,13 @@ SoftDevice Controller
...
+  If several SDUs are transmitted per ISO_Interval, meaning that the ISO_Interval is larger than the SDU_Interval, our implementation of LE Read ISO TX Sync returns a unique SDU synchronization reference per SDU.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:3230]:
```suggestion
  If several SDUs are transmitted during each ``ISO_Interval``, meaning that the ``ISO_Interval`` is larger than the ``SDU_Interval``, our implementation of the LE Read ISO TX Sync command returns a unique SDU synchronization reference for each SDU.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 5]
```diff
@@ -3222,6 +3222,13 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
+.. rst-class:: v2.6.0
+
+DRGN-21293: Le Read ISO TX Sync is implemented according to the raised errata ES-23138. 
+  In ES-23138, the return parameter TX_Time_Stamp is modified to be the SDU synchronization reference of the SDU previously scheduled for transmission, and not the CIG or BIG reference anchor point.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/known_issues.rst:3228]:
```suggestion
  In ES-23138, the return parameter ``TX_Time_Stamp`` is used as the SDU synchronization reference of the SDU previously scheduled for transmission.
  This differs from the Bluetooth Core Specification v5.4, which uses the CIG or BIG reference anchor point as the SDU synchronization reference.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by Tschet1 — APPROVED** [review]

**Comment by peknis** [comment]
Added this known issue to https://github.com/nrfconnect/sdk-nrf/pull/14185. Please close this PR.

**Pull request closed without merging**